### ISSUE TITLE
Amazon-cognito-identity-js all calls blocked by cors starting 11/15

### ISSUE THREAD

**Issue by Shadrack1701** [aws-amplify/amplify-js#7285]
All of my cognito requests for all my aws environments using Amazon-Cognito-Identity-Js are all now failing with the following:
`Access to fetch at 'https://cognito-idp.us-east-1.amazonaws.com/' from origin 'my-example.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Locally, in dev or in prod attempting to use any application operation that tries to call out to Cognito fails with this error.  (logging in, forgot password, registration etc)

I was testing and usingAccess to fetch at 'https://cognito-idp.us-east-1.amazonaws.com/' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

I actually don't believe the issue is with AWS-Amplify/Amazon-Cognito-Identity-Js.  I was more hoping to see if anyone else was experiencing this issue or if it was just me.  I only see one issue currently reported by Amazon for us-east-1 and it's with Kinesis API calls which I don't know if Cognito uses behind the scenes (I doubt it).

Here is an example of the call locally post failure (exact same issues deployed as well) for extra context:
`Request URL: https://cognito-idp.us-east-1.amazonaws.com/
Request Method: OPTIONS
Status Code: 504 Gateway Time-out
Remote Address: 3.221.103.242:443
Referrer Policy: same-origin
Connection: keep-alive
Content-Length: 534
Content-Type: text/html
Date: Wed, 25 Nov 2020 15:20:01 GMT
Server: awselb/2.0
Accept: */*
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9
Access-Control-Request-Headers: content-type,x-amz-target,x-amz-user-agent
Access-Control-Request-Method: POST
Cache-Control: max-age=0
Connection: keep-alive
Host: cognito-idp.us-east-1.amazonaws.com
Origin: http://localhost:3000
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: cross-site
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.67 Safari/537.36`

**Comment by judiel-camonapp** [comment]
https://www.reddit.com/r/aws/comments/k0sd0l/cloudwatch_useast1_problems_again/

**Comment by andriworld** [comment]
Duplicate of https://github.com/aws-amplify/amplify-js/issues/7284

**Comment by sammartinez** [comment]
@andriworld is right, I am going to close this in favor of the other issue. Please follow the above issue.
